soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -66,6 +66,19 @@ config SOC_IT8XXX2_CPU_IDLE_GATING
 	  gated by individual drivers. When this option is disabled, CPU idle
 	  mode is always permitted.
 
+config SOC_IT8XXX2_JTAG_ENABLE
+	bool "The JTAG module clock gating is configured here"
+	help
+	  This option determines if the JTAG module would be enabled or not.
+
+config SOC_IT8XXX2_ENABLE_FPU
+	bool "FPU Enable"
+	default y if FPU
+	help
+	  Enable the on-chip FPU to support the floating-point. If disabled,
+	  the FPU will be clock gated and floating-point instructions will
+	  not work because the clock would has been gated.
+
```

Comment:
Why are these changes related? Maybe expand on commit message or have it a in previous patch?

Reply:
@edersondisouza 

The clock gating is added for the power saving when the modules are not enabled/utilized in the system, however, there's an item which is related to the FPU so I tagged Peter @tari to review this part, if the option is configured, the FPU will be available because the clock gating will not be performed in this condition.

If the CONFIG_FPU is defined, the CONFIG_SOC_IT8XXX2_ENABLE_FPU will be enabled as well, then the following actions won't be issued:
`#ifndef CONFIG_SOC_IT8XXX2_ENABLE_FPU
    /* FPU Clock Gating */
    ecpm_regs->CGCTRL6R |= FPU_CHA_CLOCK_GATING;
#endif`